lib/brcobranca.rb

```diff
@@ -117,9 +117,14 @@ module Template
   module Retorno
     autoload :Base,            'brcobranca/retorno/base'
     autoload :RetornoCbr643,   'brcobranca/retorno/retorno_cbr643'
-    autoload :RetornoCnab240,  'brcobranca/retorno/retorno_cnab240'
+    autoload :RetornoCnab240,  'brcobranca/retorno/retorno_cnab240' # DEPRECATED
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.